### ISSUE TITLE
Missing internal transaction entries for `EthereumTransactionData` interface

### ISSUE THREAD

**Issue by Shadowfiend** [blocknative/sdk#138]
According to https://docs.blocknative.com/notify-sdk#internal-transactions-ethereum (and experimentally borne out in practice), there are two keys, `internalTransactions` and `netBalanceChanges`, available on `EthereumTransactionData`. The shape of `netBalanceChanges` differs from the one described in `BitcoinTransactionData`:

https://github.com/blocknative/sdk/blob/a5c9f5314323f2556fc864eff6e3dc32205d3c2d/src/interfaces.ts#L40

Instead, it's more like:

```typescript
  netBalanceChanges: EthereumNetBalanceChanges[]
  internalTransactions: EthereumInternalTransaction[]
…

type EthereumNetBalanceChanges = {
  address: string
  balanceChanges: EthereumAssetBalanceChanges[]
}

type EthereumAssetBalanceChanges = {
  delta: string
  asset: AssetDetails
  breakdown: TransferDetails[]
}

type AssetDetails = {
  type: AssetType
  symbol: string
}

type AssetType = "ether" | "ERC20" | ...

type TransferDetails = {
  counterparty: string
  amount: string
}

type EthereumInternalTransaction = {
  // ... some details
}
```

Any chance we can get these added?

**Comment by Shadowfiend** [comment]
Actually, the docs are also wrong, sorry. I've updated above to indicate the correct type structure.

**Comment by aaronbarnardsound** [comment]
Thanks @Shadowfiend  I will take a look at these and get them fixed up

**Comment by aaronbarnardsound** [comment]
Fixed via #140
